PureBasicIDE/Language.pb

```diff
@@ -752,6 +752,7 @@ DataSection
   Data$ "Email",            "Email"
   Data$ "EnableMenuIcons",  "Display Icons in the Menu"
   Data$ "DisplayFullPath",  "Display full Source Path in TitleBar"
+  Data$ "DisplayDarkMode",  "Display Enable DarkMode"
```

Comment:
Not sure what "Display Enable DarkMode" is supposed to mean. "Enable Dark Mode" sounds more right to me.

Reply:
Thats right. 'Enable Dark Mode' is good

Reply:
Please do not forget that the PB Library 'StatusBar' from version v5.71 and higher is not yet fixed. The background colour does not work. Therefore I use the 'StatusBar' library from version v5.70 (ASM).

Reply:
Bugreport macOS StatusBar: https://www.purebasic.fr/english/viewtopic.php?t=76728